eo-parser/src/main/antlr4/org/eolang/parser/Program.g4

```diff
@@ -217,6 +219,7 @@ META: PLUS NAME (SPACE ~[\r\n]+)?;
 REGEX: SLASH ~[\r\n]+ SLASH [a-z]*;
 
 ROOT: 'Q';
+STANDARD_ROOT: 'QQ';
```

Comment:
@yegor256 Is this name ok?

Reply:
@Graur yeah, sounds right, but I would name it with a single noun, maybe: `HOME`. see https://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html